src/main/java/org/spring/FullVersion/FamilyService.java

```diff
@@ -78,10 +81,20 @@ public Family adoptChild(Family family, Human child) {
     }
 
     public void deleteAllChildrenOlderThen(int age) {
-        for (Family family : familyDao.getAllFamilies()) {
-            family.getChildren().removeIf(child -> Integer.parseInt(child.describeAge()) > age);
-            familyDao.saveFamily(family);
-        }
+        LocalDate currentDate = LocalDate.now();
+
+        familyDao.getAllFamilies().stream()
+                .peek(family -> family.getChildren().removeIf(child -> {
+                    LocalDate birthDate = convertToLocalDate(child.describeAge());
```

Comment:
instead of writing this code here you can implement getAge method on Human class

Reply:
like child.getAge? @sabuhihuseynov